### PR TITLE
modify Sidebar design and implement responsive design

### PR DESCRIPTION
**Pull request by queq1890**
## Story URL
https://trello.com/c/6Zj9ZmdE/12-5-fix-sidebar-ui-main-ui-design

## What I did
- Fix Sidebar design 
  - Make profile pic size smaller,
  - Change "edit profile" link position
- Add responsive design
  - Use `felx-box` instead of `float`
  - Change `font-size` definition ( px => rem )
  - Add media query at some widths

## Something I have to implement later
- Add Sidebar ( or Topbar ) for SmartPhone

### CODE REVIEW

File: src/containers/App/index.js [hunk 1]
```diff
@@ -26,8 +26,8 @@ const App = () => (
     <MainWrapper>
       <Switch>
         <Route exact path="/" component={Home} />
-        <Route exact path="/auth" component={Auth} />
-        <Route exact path="/daily_reports" component={DailyReports} />
+        <Route path="/auth" component={Auth} />
+        <Route path="/Member" location={location} component={Member} />
```
Comment (ggianglv) [inline src/containers/App/index.js]: i think path="/Member" should be path="/member"

Comment (phuongthuan) [inline src/containers/App/index.js]: The route path should be lowercase. 
`<Route path="/member" location={location} component={Member} />`

File: src/containers/Member/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React, { Fragment } from 'react';
+import { Route, Switch } from 'react-router-dom';
+
+import MemberSidebar from 'components/MemberSidebar';
+import EditProfile from './EditProfile';
+import DailyReports from './DailyReports';
+import NewDailyReport from './NewDailyReport';
+
+const Member = () => {
+  const pathName = location.pathname;
```
Comment (phuongthuan) [inline src/containers/Member/index.js]: Should  be destructing this variable:
`const Member = ({ pathname }) => {`

And if you do this, you don't need to`return` keyword. 
If no logic in this component, pls using stateless component instead.
Like this: 


```
const NewDailyReport = () => (
  <ContentWrapper>
    <ContentHeader>Write a Daily Report</ContentHeader>
  </ContentWrapper>
);
```

**Review by phuongthuan — APPROVED** [review]

**Comment by queq1890** [comment]
Fixed and  Merging. Thank you guys!

**Pull request merged**